_posts/courses/intermediate-earth-data-science-textbook/01-time-series/plot-time-series-handle-dates/2019-11-19-time-series-01-get-started-with-time-series-python.md

```diff
@@ -270,12 +270,19 @@ boulder_precip_2003_2013.plot(x="DATE",
 plt.show()
 ```
 
+{:.output}
+    /opt/conda/lib/python3.8/site-packages/pandas/plotting/_matplotlib/core.py:1235: UserWarning: FixedFormatter should only be used together with FixedLocator
```

Comment:
@nkorinek looks like we may want to turn off warnings or fix this line as this will print to the website

Reply:
ok I can fix that!